x-pack/plugins/triggers_actions_ui/public/application/sections/common/components/execution_duration_chart.tsx

```diff
@@ -81,20 +81,34 @@ export const ExecutionDurationChart: React.FunctionComponent<ComponentOpts> = ({
               }}
             />
             <BarSeries
-              id="executionDuration"
+              id={i18n.translate(
+                'xpack.triggersActionsUI.sections.executionDurationChart.durationLabel',
+                {
+                  defaultMessage: `Duration`,
+                }
+              )}
               xScaleType="linear"
               yScaleType="linear"
               xAccessor={0}
               yAccessors={[1]}
               data={paddedExecutionDurations.map((val, ndx) => [ndx, val])}
+              minBarHeight={2}
             />
             <LineSeries
-              id="rule_duration_avg"
+              id={i18n.translate(
```

Comment:
I'm not sure that id is the right way to set the title, `LineSeries` has `name` property which could be better here to set

Reply:
Updated! I found the elastic charts documentation a little hard to understand so thank you!